lib/Doctrine/ORM/EntityManager.php

```diff
@@ -953,6 +953,14 @@ public function initializeObject($obj)
         $this->unitOfWork->initializeObject($obj);
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public function isUninitializedEntity($obj)
```

Comment:
you should add the parameter and return types

Reply:
return type done, `mixed` for param is not compatible with PHP 7.1. `@inheritDoc` to the rescue, from https://github.com/doctrine/persistence/pull/334